src/main/resources/alma/fix/item.fix

```diff
@@ -24,7 +24,7 @@ do list(path: "HOL  ", "var": "$i")
     unless in("$i.8", "ITM  .*.H")
       add_field( "hasItem[].$append.test","")
       add_field("hasItem[].$last.label", "lobid Bestandsressource")
-      set_array("hasItem[].$last.type[]", "Item","NoItem")
+      set_array("hasItem[].$last.type[]", "Item","PhysikalischerTitel")
       copy_field("$i.8", "hasItem[].$last.heldBy.id")
```

Comment:
I know, this is preliminary, and my pedantry my be annoying  - but I think it would be nonetheless better to use also an English nomen to not mix languages.